docs/source/orderer/ordering_service.md

```diff
@@ -135,36 +143,35 @@ packaged into multiple different blocks that compete to form a chain.
 In Hyperledger Fabric, the blocks generated by the ordering service are
 **final**. Once a transaction has been written to a block, its position in the
 ledger is immutably assured. As we said earlier, Hyperledger Fabric's finality
-means that there are no **ledger forks** --- validated transactions will never
-be reverted or dropped.
+means that there are no **ledger forks** --- validated and committed transactions
+will never be reverted or dropped.
 
 We can also see that, whereas peers execute smart contracts and process transactions,
 orderers most definitely do not. Every authorized transaction that arrives at an
-orderer is mechanically packaged in a block --- the orderer makes no judgement
+orderer is then mechanically packaged into a block --- the orderer makes no judgement
 as to the content of a transaction (except for channel configuration transactions,
 as mentioned earlier).
 
 At the end of phase two, we see that orderers have been responsible for the simple
 but vital processes of collecting proposed transaction updates, ordering them,
-and packaging them into blocks, ready for distribution.
+and packaging them into blocks, ready for distribution to the gateway service.
 
-### Phase three: Validation and commit
+### Phase three: Validation and commitment
 
-The third phase of the transaction workflow involves the distribution and
-subsequent validation of blocks from the orderer to the peers, where they can be
-committed to the ledger.
+The third phase of the transaction workflow involves the distribution of
+ordered and packaged blocks from the ordering service to the gateway service,
+where peers validate the transactions before committing them to the ledger.
 
-Phase 3 begins with the orderer distributing blocks to all peers connected to
-it. It's also worth noting that not every peer needs to be connected to an orderer ---
+Phase 3 begins with the orderer distributing blocks to the gateway. It's
```

Comment:
We should probably include the recommendation that all peers get blocks from the ordering service in this section.

Reply:
Well we shouldn't really say you can use gossip but don't use it. 
Any recommendation should indicate what the consequences of the choice are, such as:
 - get blocks directly from ordering service in order to ... 
-  or alternatively, use gossip in order to ...

Reply:
Added the recommendation without why it matters text.